gunicorn/arbiter.py

```diff
@@ -522,37 +522,40 @@ def reap_workers(self):
                     # A worker was terminated. If the termination reason was
                     # that it could not boot, we'll shut it down to avoid
                     # infinite start/stop cycles.
-                    exitcode = status >> 8
+
+                    exitcode = os.WEXITSTATUS(status) \
+                        if os.WIFEXITED(status) else 0
+
+                    signum = os.WTERMSIG(status) \
+                        if os.WIFSIGNALED(status) else 0
+                    try:
+                        sig_name = signal.Signals(signum).name
+                    except ValueError:
+                        sig_name = str(signum)
+
                     if exitcode != 0:
-                        self.log.error('Worker (pid:%s) exited with code %s', wpid, exitcode)
+                        self.log.error("Child exited with code %s (pid: %s)",
```

Comment:
we should use "Worker"  instead of "Child" here. This is more meaningful in the context of Gunicorn arbitration.

Reply:
Hi Benoit, 

Sorry, I totally missed that I committed to this open PR! That was not my intention. I'll have a look what happened before and what I want from these changes. My intent was to lower the log level on the regular reload/sigterms. And the two raises in between the log statements were irksome to me. 

What I find confusing that both master and worker are logging the same event, with the same wording. In the logs it's not clear which process is which.